### ISSUE TITLE
TS node process out of memory

### ISSUE THREAD

**Issue by dbmeneses** [SonarSource/SonarTS#598]
When trying to do a minimal test of SonarTS embedded in SonarLint Core in Appveyor, it fails to start the SonarTS Server:
https://ci.appveyor.com/project/SonarSource/sonarlint-core/build/1.0.1316#L2476

**SonarTS version:** 1.5.0.2122<br>
**TypeScript version:** 2.6.1<br>

It could be caused by SonarTS forking a TS node with a high memory allocation (2GB of memory, maybe?) and Appveyors not having enough memory available. The same build works fine on Linux with Travis.

**Reproducer** 
Sonarlint-core unit tests on Appveyor consistently reproduce the problem.
If the cause given above is correct, any execution of SonarTS should trigger this problem if the machine doesn't have much memory, independently of what is analyzed.

**Logs**
```
`eploying bundle from `/sonarts-bundle.zip` to `C:\Users\appveyor\AppData\Local\Temp\1\junit4413676563094099971\junit6226299006239531974\work\.sonartmp_9137099978356163663\2697142273491232957`
FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - process out of memory
Failed to start SonarTS Server`

**Comment by saberduck** [comment]
@dbmeneses do you know the version of NodeJS installed?

**Comment by dbmeneses** [comment]
No, but should be the latest one: https://www.appveyor.com/docs/lang/nodejs-iojs/

**Comment by vilchik-elena** [comment]
https://jira.sonarsource.com/browse/MMF-1562
